integration/docker/entrypoint.sh

```diff
@@ -15,7 +15,6 @@ set -e
 ALLUXIO_HOME="/opt/alluxio"
 NO_FORMAT='--no-format'
 FUSE_OPTS='--fuse-opts'
-MOUNT_POINT='/mnt/alluxio-fuse'
```

Comment:
I think we can do this in one line 

```suggestion
 MOUNT_POINT="${MOUNT_POINT:-/mnt/alluxio-fuse}"
```

Reply:
Thanks for your suggestion. Done